Sources/ClientRuntime/Serialization/SerializationUtils/XML/ValueWriters.swift

```diff
@@ -5,9 +5,9 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-import class SmithyXML.Writer
+import protocol SmithyReadWrite.SmithyWriter
 
-extension Writer {
+extension SmithyWriter {
```

Comment:
This now extends the `SmithyWriter` protocol instead of the concrete XML writer class.